internal/driver/glfw/window_test.go

```diff
@@ -583,7 +583,6 @@ func TestWindow_Scrolled(t *testing.T) {
 
 	w.mousePos = fyne.NewPos(50, 60)
 	w.mouseScrolled(w.viewport, 10, 10)
-	w.waitForEvents()
```

Comment:
I removed this, because `mouseScrolled` is a sync function, so `waitForEvents` does not do anything, IMHO.